accelerator/main.c

```diff
@@ -73,9 +73,7 @@ int main(int argc, char* argv[]) {
   }
 
   log_warning(logger_id, "Tangle-accelerator starts running\n");
-  while (ta_http.running) {
-    ;
-  }
+  getchar();
```

Comment:
Can you replace `getchar` with other lightweight calls such as [sched_yield](http://man7.org/linux/man-pages/man2/sched_yield.2.html)?